util/dvsim.py

```diff
@@ -458,10 +458,14 @@ def main():
         cfg.deploy_objects()
 
         # Generate results.
-        results = cfg.gen_results()
+        cfg.gen_results()
+        cfg.gen_results_summary()
```

Comment:
can we call this from within `gen_results` directly? this eliminates the two calls to similar functions here, and we need to check whether this is a master config anyways inside one of the two `gen_results*` functions...

Reply:
sure, I can move it to gen_result.